tests/unit/technical_indicators.py

```diff
@@ -0,0 +1,570 @@
+"""
```

Comment:
This file "technical_indicators.py" does not seem to belong in the unit tests. Isn't it a re-implementation or similar to "gquant/cuindicator/indicator.py" module? You should just import and use "indicator.py" module.

Reply:
It was the original Pandas technical indicator implementation from:
[pandas-technical-indicators](https://github.com/Crypto-toolbox/pandas-technical-indicators)
I used it as a reference to verify the results of my customized GPU kernel implementation.

Reply:
And it should just be part of the test module

Reply:
I see, you're testing  "gquant/cuindicator/indicator.py" against "technical_indicators.py". Makes sense then.